launcher/ui/MainWindow.cpp

```diff
@@ -1955,6 +1979,7 @@ void MainWindow::on_actionDeleteInstance_triggered()
     if (response == QMessageBox::Yes)
     {
         APPLICATION->instances()->deleteInstance(id);
+        ui->actionUndoTrashInstance->setEnabled(true);
```

Comment:
There is nothing to undo if the instance is permanently deleted (no trash).

Reply:
fixed